app/components/admin/navigation_component.rb

```diff
@@ -3,13 +3,20 @@
 module Admin
   # Displays the drop down navigation for the admin pages
   class NavigationComponent < ApplicationComponent
+    def initialize(selected: nil)
+      @selected = selected
+    end
+
+    attr_reader :selected
+
     def dropdown
```

Comment:
this `selected` value allows us to choose a different item in the admin selection drop-down menu to be selected from the page currently being rendered (which is useful in this PR)